shoop/core/models/products.py

```diff
@@ -258,20 +258,29 @@ def get_cheapest_child_price(self, context, quantity=1):
             for c in self.variation_children.all()
         )[0]
 
-    def get_price(self, context, quantity=1):
+    def get_price_info(self, context, quantity=1):
```

Comment:
Could we get proper docstrings for `get_price_info`, `get_price` and `get_base_price`?

I mean, I'm not sure about the semantics of the last one, at least :)